client_test.go

```diff
@@ -857,14 +858,27 @@ func TestMessages(t *testing.T) {
 		})
 
 		Convey("we can publish a multipart messages", func() {
+			fileName := "cat.jpg"
+			file, err := os.Open(fileName)
```

Comment:
Convey is going to pass by here a lot of times, we should probably close this handle (even though it's just a test)